app/src/main/java/com/android/sample/model/image/ImageRepositoryFirestore.kt

```diff
@@ -8,9 +8,11 @@ import com.google.firebase.storage.FirebaseStorage
 import java.io.ByteArrayOutputStream
 import javax.inject.Inject
 
-open class ImageRepositoryFirestore @Inject constructor(private val firestore: FirebaseFirestore) :
+open class ImageRepositoryFirestore
```

Comment:
included the addition of dependency injection for FirebaseStorage and the use of a compression quality of 50 for images to reduce their size before uploading to Firestore. These enhancements streamline FirebaseStorage usage and optimize image storage. LGTM!